kubernetes/samples/scripts/create-weblogic-domain/model-in-image/model1.yaml.jrf

```diff
@@ -0,0 +1,35 @@
+domainInfo:
+    AdminUserName: '@@FILE:/weblogic-operator/secrets/username@@'
+    AdminPassword: '@@FILE:/weblogic-operator/secrets/password@@'
+    ServerStartMode: 'prod'
+    RCUDbInfo:
+        rcu_prefix: '@@FILE:/weblogic-operator/config-overrides-secrets/sample-domain1-rcu-access/rcu_prefix@@'
+        rcu_schema_password: '@@FILE:/weblogic-operator/config-overrides-secrets/sample-domain1-rcu-access/rcu_schema_password@@'
+        rcu_admin_password: '@@FILE:/weblogic-operator/config-overrides-secrets/sample-domain1-rcu-access/rcu_admin_password@@'
```

Comment:
rcu_admin_password is not a required attribute since we are not supporting creation of rcu schema on the fly by WDT.

Reply:
change complete - sample doc updated to - usage doc will be updated by Rosemary in her branch